cypress/integration/login.js

```diff
@@ -8,20 +8,87 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Check login as Admin and if the day is today', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Edvinas Kveselys"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Admin"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');

```

Comment:
I don't think we nee this block, because it seems like we're already asserting we're Admins later on